### PR TITLE
added ui change for possible payout

### PR DESCRIPTION
**Pull request by tricelex**
added ui change for possible payout #120

### CODE REVIEW

File: packages/react-app/src/routes/party/components/ViewTable.jsx [hunk 1]
```diff
@@ -39,14 +39,17 @@ export const ViewTable = ({ partyData, mainnetProvider, votesData, distribution,
               <Td>
                 <Center>{!isNaN(dist[d] * 1) && dist && (dist[d] * 100).toFixed(2)}%</Center>
               </Td>
+              <Td>
+                <Center>{!isNaN(dist[d]) ? (dist[d] * amountToDistribute).toFixed(2) : amountToDistribute}</Center>
```
Comment (hansmrtn) [inline packages/react-app/src/routes/party/components/ViewTable.jsx]: I'd like to see this column hid if the amountToDistribute is nil. Always showing "possible payout" is misleading until an actual distribution is taking place.

File: packages/react-app/src/routes/party/components/ViewTable.jsx [hunk 2]
```diff
@@ -62,6 +65,9 @@ export const ViewTable = ({ partyData, mainnetProvider, votesData, distribution,
             <Th>
               <Center>{`Score (${strategy})`}</Center>
             </Th>
+            <Th>
+              <Center>{"possible payout"}</Center>
```
Comment (hansmrtn) [inline packages/react-app/src/routes/party/components/ViewTable.jsx]: Consider renaming to just "Payout" as the user should assume that what is being shown is accurate.

**Review by hansmrtn — CHANGES_REQUESTED** [review]

File: packages/react-app/src/routes/party/components/ViewTable.jsx (hunk 1 above)
Comment (tricelex) [reply packages/react-app/src/routes/party/components/ViewTable.jsx]: fixed in latest commit.

File: packages/react-app/src/routes/party/components/ViewTable.jsx (hunk 2 above)
Comment (tricelex) [reply packages/react-app/src/routes/party/components/ViewTable.jsx]: renamed in latest commit.

**Review by hansmrtn — APPROVED** [review]

**Pull request merged**